wagtail/wagtailsearch/backends/elasticsearch.py

```diff
@@ -26,7 +26,7 @@ class ElasticsearchMapping(object):
         'DateField': 'date',
         'DateTimeField': 'date',
         'DecimalField': 'double',
-        'FileField': 'string',
+        'FileField': 'attachment',
```

Comment:
That's fine with me. We should definitely make this opt-in as I imagine this would make indexing large PDFs take much longer (and may time out the upload request).